src/NuGet.Core/NuGet.Build.Tasks/NuGet.RestoreEx.targets

```diff
@@ -32,6 +32,7 @@ Copyright (c) .NET Foundation. All rights reserved.
         SolutionPath="$(SolutionPath)"
         ProcessFileName="$(NuGetConsoleProcessFileName)"
         MSBuildStartupDirectory="$(MSBuildStartupDirectory)"
+        BinlogPath="$(RestoreBinlogPath)"
```

Comment:
It might be worth naming this `RestoreStaticGraphBinLogPath` so its obvious it only works for static graph-based restore.

Reply:
Thanks, great feedback. Updated the PR.